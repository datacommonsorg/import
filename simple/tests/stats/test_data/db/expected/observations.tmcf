Node: E:Table->E0
typeOf: dcs:StatVarObservation
variableMeasured: C:Table->variable
observationDate: C:Table->date
observationAbout: C:Table->entity
value: C:Table->value