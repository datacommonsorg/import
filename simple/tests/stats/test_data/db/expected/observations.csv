entity,variable,date,value
e1,v1,2023,123
e2,v1,2023,456
